packages/portal/src/mixins/advancedSearch.js

```diff
@@ -12,7 +12,7 @@ const FIELD_SUGGEST_CONCEPT = 'concept';
 const FIELD_SUGGEST_PLACE = 'place';
 
 const advancedSearchFields = [
-  { name: 'fulltext', type: FIELD_TYPE_FULLTEXT },
+  { name: 'fulltext', type: FIELD_TYPE_FULLTEXT, supportsExact: true },
```

Comment:
Instead of adding supportsExact to specific fields, derive it from the fact the the field type has a query property in the exact modifier.